openedx/core/djangoapps/courseware_api/tests/test_views.py

```diff
@@ -427,6 +427,15 @@ def test_resume_with_completion(self):
         assert response.data['unit_id'] == str(self.unit.location)
         assert response.data['section_id'] == str(self.sequence.location)
 
+    def test_resume_invalid_key(self):
```

Comment:
Docstring maybe? I don't really know what this is doing here.

Reply:
Done.